src/deidservice/azext_deidservice/aaz/latest/deidservice/_create.py

```diff
@@ -0,0 +1,440 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "deidservice create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Create a DeidService
+
+    :example: DeidServices_Create - generated by [MaximumSet] rule - generated by [MaximumSet] rule
```

Comment:
You can remove the duplicated suffix
```suggestion
    :example: DeidServices_Create
```

Reply:
fixed